app/employee-app/backend/types.bal

```diff
@@ -15,13 +15,22 @@
 // under the License.
 import employee_service.database;
 
-# Employee type.
+# Employee record.
 public type Employee record {|
     *database:Employee;
 |};
 
-
 # Employee filter record.
 public type EmployeeFilter record {|
     *database:EmployeeFilter;
 |};
+
+# Organization data filter record.
+public type OrgDetailsFilter record {|
```

Comment:
Shall we rename this to `orgStructureFilter`. then the turm will be more relevant to the resource function definition.